### PR TITLE
adding graphing files to cryptocoin

### PR DESCRIPTION
**Pull request by nj-io**
please check commit log for details on commit

### CODE REVIEW

**Review by MarlenAw — APPROVED** [review]

**Review by hasansaad1 — APPROVED** [review]
we might have some conflicts with the other branch

File: package.json [hunk 1]
```diff
@@ -21,23 +25,27 @@
...
+    "css-loader": "^0.28.9",
     "eslint": "^4.15.0",
     "eslint-config-airbnb": "^16.1.0",
     "eslint-config-prettier": "^2.9.0",
+    "eslint-loader": "^1.9.0",
     "eslint-plugin-import": "^2.8.0",
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^2.4.0",
     "eslint-plugin-react": "^7.5.1",
+    "npm-check": "^5.5.2",
```
Comment (Karyum) [inline package.json:37]: what does this module do ?

File: package.json [hunk 2]
```diff
@@ -21,23 +25,27 @@
...
+    "postcss-loader": "^2.0.10",
```
Comment (Karyum) [inline package.json:38]: also what does this module do ?

**Review by Karyum — COMMENTED** [review]
@ionush im a bit afraid of contiue to review this pr cause i feel like #14 will cause you a lot of merge conflicts.

i'm not going to review it cause it's just pointless im gonna have to re review it after the merge conflicts resolved

what do you think?

File: package.json (hunk 1 above)
Comment (nj-io) [reply package.json:37]: checks whether the modules are being used in code and if there are updates. pretty useful

File: package.json (hunk 2 above)
Comment (nj-io) [reply package.json:38]: allows the css to be loaded in webpack

**Comment by nj-io** [comment]
i'm closing because the redux version has all of the graphing already
@Karyum please qa that

**Pull request closed without merging**